fsutils/fsutils_test.go

```
@@ -0,0 +1,393 @@
+
+	t.Run("permission denied", func(t *testing.T) {
+		tempDir, err := os.MkdirTemp("", "testdir")
+		if err != nil {
+			t.Fatal(err)
+		}
+		defer os.RemoveAll(tempDir)
+
+		if err := os.Chmod(tempDir, 0000); err != nil {
+			t.Fatal(err)
+		}
+		defer os.Chmod(tempDir, 0755)
```

golangci-lint (1.62.2)

Error return value of `os.Chmod` is not checked

(errcheck)